tools/installers/install_whisper.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+MAKE=make
+
+if [ $# != 0 ]; then
+    echo "Usage: $0"
+    exit 1;
+fi
+
+if ! python -c "import packaging.version" &> /dev/null; then
+    python3 -m pip install packaging
+fi
+torch_110_plus=$(python3 <<EOF
+from packaging.version import parse as V
+import torch
+
+if V(torch.__version__) >= V("1.10.1"):
+    print("true")
+else:
+    print("false")
+EOF
+)
+
+python_37_plus=$(python3 <<EOF
+from packaging.version import parse as V
+import sys
+
+if V("{}.{}.{}".format(*sys.version_info[:3])) >= V("3.6"):
+    print("true")
+else:
+    print("false")
+EOF
+)
+
+cuda_version=$(python3 <<EOF
+import torch
+if torch.cuda.is_available():
+    version=torch.version.cuda.split(".")
+    # 10.1.aa -> 101
+    print(version[0] + version[1])
+else:
+    print("")
+EOF
+)
+echo "cuda_version=${cuda_version}"
+
```

Comment:
Can you leave a note about the git commit hash info in case it breaks sometime in the future?

Reply:
Ok, just add commit hash as a comment?

Reply:
I think I'll just upload training config for Whisper frontend, as I already did

Reply:
yeah, just commit hash as a comment.
Sorry, I mean the results of this config on CHiME4 and Librispeech_100 which you are training now, for a record.

Reply:
Oh, I see. I'll add them to the recipe READMEs. Does this sound good?